SPECS/xcp-ng-xapi-storage.spec

```diff
@@ -79,6 +78,8 @@ cd build
 %{_docdir}/xcp-ng-xapi-storage/
 %{_prefix}/lib/python2.7/site-packages/xapi/storage/libs/
 %{_prefix}/lib/python2.7/site-packages/xcp_ng_xapi_storage_libs-*-py2.7.egg-info
+%exclude %{_prefix}/lib/python2.7/site-packages/xapi/storage/libs/libcow/qcow2util.py
+%exclude %{_prefix}/lib/python2.7/site-packages/xapi/storage/libs/qemudisk.py
```

Comment:
I'd add a comment explaining why (I know it's already in the changelog, but it saves time when it's right in front of your eyes)